server/command.go

```diff
@@ -186,39 +188,71 @@ type EmptyReplicationHandler struct {
 }
 
 func (h EmptyHandler) UseDB(dbName string) error {
+	log.Printf("Received: UseDB %s", dbName)
 	return nil
 }
 func (h EmptyHandler) HandleQuery(query string) (*Result, error) {
+	log.Printf("Received: Query: %s", query)
+
+	// These two queries are implemented for minimal support for MySQL Shell
+	if query == `SET NAMES 'utf8mb4';` {
+		return nil, nil
+	}
+	if query == `select concat(@@version, ' ', @@version_comment)` {
+		r, err := mysql.BuildSimpleResultset([]string{"concat(@@version, ' ', @@version_comment)"}, [][]interface{}{
+			{"8.0.11"},
+		}, false)
+		if err != nil {
+			return nil, err
+		} else {
+			return &mysql.Result{
+				Status:       0,
+				Warnings:     0,
+				InsertId:     0,
+				AffectedRows: 0,
+				Resultset:    r,
+			}, nil
+		}
+	}
+
 	return nil, fmt.Errorf("not supported now")
 }
 
 func (h EmptyHandler) HandleFieldList(table string, fieldWildcard string) ([]*Field, error) {
+	log.Println("Received: FieldList")
```

Comment:
It's helpful to print the function arguments? Also for below methods

Reply:
Done.